src/less/pagination/ds6/pagination-base.less

```diff
@@ -0,0 +1,104 @@
+.pagination {
+    font-size: 0;
+    min-width: 400px;
+    text-align: center;
+    a {
+        font-size: 16px;
+    }
+}
+
+.pagination__items {
+    display: inline-block;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    vertical-align: middle;
+}
+
+.pagination__next,
+.pagination__previous {
+    display: inline-block;
+    fill: @ds6-color-text-default;
+    font-size: 19px;
+    line-height: 48px;
+    margin-right: 8px;
+    width: 48px;
+    text-align: center;
+    vertical-align: middle;
+
+    &:visited {
+        fill: @ds6-color-text-default;
+    }
+
+    &:focus,
+    &:hover {
+        fill:  @ds6-color-link-default;
```

Comment:
Additional space here causing lint warning.

Reply:
fixed in the next commit :)